packages/legacy-events/SyntheticEvent.js

```diff
@@ -78,6 +78,7 @@ function SyntheticEvent(
   this.nativeEvent = nativeEvent;
   this._dispatchListeners = null;
   this._dispatchInstances = null;
+  this._dispatchContainers = null;
```

Comment:
This also needs to be cleared in `destructor`.  Otherwise we're risking a memory leak.